rules/windows/builtin/win_net_use_admin_share.yml

```diff
@@ -0,0 +1,26 @@
+title: Mounted Windows Admin Shares with net.exe
+id: 3abd6094-7027-475f-9630-8ab9be7b9725
+status: experimental
+description: Detects when an admin share is mounted using net.exe
+references:
+    - https://drive.google.com/file/d/1lKya3_mLnR3UQuCoiYruO3qgu052_iS_/view
+author: Teymur Kheirkhabarov '@HeirhabarovT', Zach Stanford '@svch0st'
```

Comment:
Hello @svch0stz !

Could you please put the quotes only once, on both ends of the author field? that's a common style, I am not sure if it will break something, but still: 

```
 grep -irh 'author: ' ./ | grep -e '"' -e "'" | sort | uniq
author: "@BarryShooshooga"
author: "@SBousseaden, Florian Roth"
author: "Dimitrios Slamaris, @atc_project (fix)"
author: "Lionel PRAT, Christophe BROCAS, @atc_project (improvements)"
author: '@2xxeformyshirt (Security Risk Advisors) - rule; Teymur Kheirkhabarov (idea)'
author: '@41thexplorer, Microsoft Defender ATP'
author: '@ROxPinTeddy'
author: '@SBousseaden (detection), Thomas Patzke (rule)'
author: '@caliskanfurkan_'
author: '@neu5ron'
author: '@neu5ron, Florian Roth'
author: '@neu5ron, SOC Prime'
author: '@neu5ron, Teymur Kheirkhabarov, oscd.community'
author: 'Furkan Caliskan (@caliskanfurkan_)'
author: 'Josh Brower @DefensiveDepth'
author: 'Nate Guagenti (@neu5ron), Open Threat Research (OTR)'
author: 'SOC Prime, Adam Swan'
author: 'Samir Bousseaden, @neu5rn'
author: 'Samir Bousseaden, @neu5ron'
```